internal/cli/context_create.go

```diff
@@ -51,9 +58,10 @@ func (c *ContextCreateCommand) Flags() *flag.Sets {
 	return c.flagSet(0, func(set *flag.Sets) {
 		f := set.NewSet("Command Options")
 		f.BoolVar(&flag.BoolVar{
-			Name:   "set-default",
-			Target: &c.flagSetDefault,
-			Usage:  "Set this context as the new default for the CLI.",
+			Name:    "set-default",
+			Target:  &c.flagSetDefault,
+			Default: true,
+			Usage:   "Set this context as the new default for the CLI. Defaults to true.",
```

Comment:
BTW we don't need the `Defaults to true` in the doc string since the CLI adds this automatically:

```
brian@localghost:waypoint λ waypoint context create -help                                                                                                                                                                                                                                                             
Command Options:
...
...

  -set-default
      Set this context as the new default for the CLI. Defaults to true. The
      default is true.
```

Reply:
I can make a PR to fix this real quick

Reply:
https://github.com/hashicorp/waypoint/pull/3048

Reply:
Oh ha, I was fixing the website docs. Looks like the CLI prints the default, but the website docs don't?

Reply:
Agree that the cli help text is more important than the website docs, so I support reverting this for now.